### DISCUSSION TITLE
Problems ssh to windows only from homebridge terminal

### DISCUSSION THREAD

**Discussion by ghost** [AlexGustafsson/homebridge-wol#258] [General]

**Comment by AlexGustafsson** [comment]
When you say type your password, is that when you try to use the key? A key can be created either with encryption by specifying a password or without by not specifying a password. If a password is selected, it will have to be manually entered every time the key is used. I doubt, however, that you would get "Permission denied" in that case.

I think this has to do with file permissions. The user running the homebridge process doesn't seem to have the right to access the file. You'll have to identify what user homebridge is running as (should probably be documented somewhere). You can for example try to run `ps` on your docker host. You should be able to see `homebridge` listed there somewhere, along with a user id and pid. You can also check the file permission of the other files created by homebridge. The file permissions of the key must match those.
